src/keys-builder/utils/scope.utils.ts

```diff
@@ -15,3 +15,8 @@ export function getScopes() {
 export function hasScope(scope: string) {
   return scopeToAlias.hasOwnProperty(scope);
 }
+
+export function resetScopes() {
+  scopeToAlias = {};
+  aliasToScope = {};
+}
```

Comment:
Correct me if I'm wrong, this is something related just to tests right?
Also, what was the case that made you add this?

Reply:
If it's only test-related please resolve this from within the test's scope, this shouldn't be part of the source code.

Reply:
Honestly, I can't think of a way to resolve this within the test's scope. We need to reset the scopes and the only way I can think of is to expose this method for doing so.

We could mark this method as a method only meant for internal use or something similar.

Unless you have some other idea maybe...?

Reply:
It's ok for since we aren't going to expose this 👍